docs/2.7.0/docs/canton/usermanual/security.rst

```diff
@@ -550,6 +550,24 @@ AWS KMS only supports the :ref:`following cryptographic schemes <canton_supporte
     provider it can no longer be reverted without a full reset of the node
     (i.e., re-generation of node identity and all keys).
 
+In the previous example, Canton creates its own keys on startup. If we want to create the keys ourselves and pass then
+to Canton we have to manually start the nodes by adding the following flag in the config:
+
+.. code-block:: none
+
+    <node>.init.auto-init = false
+
+We then need to register the keys in Canton by running the key registration command on each node.
+For example for a participant we would run:
```

Comment:
please document what `namespaceKmsKeyId` and friends needs to be, e.g., a AWS KMS Key ARN